src/elasticsearch/elasticsearchutils.py

```diff
@@ -31,7 +32,10 @@
   }
  },
  "script" : {
-  "inline" : "ctx._source.status='<STATUS>' ; ctx._source.modified_at='<MODIFIED_AT>'"
+  "params" : {
```

Comment:
Can you please remove all formate change or unnecessary change.  Did you add test case ?

Reply:
I havent added any new test case. These are refactoring code changes from dev branch.